packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -91,19 +91,34 @@ enum MainAxisAlignment {
 
 /// How the children should be placed along the cross axis in a flex layout.
 enum CrossAxisAlignment {
-  /// Place the children as close to the start of the cross axis as possible.
+  /// Place the children with their start edge aligned with the start side of
+  /// the cross axis.
+  ///
+  /// For example, in a column (a flex with a vertical axis), this aligns the
+  /// left edge of the children along the left edge of the column.
   start,
 
   /// Place the children as close to the end of the cross axis as possible.
+  ///
+  /// For example, in a column (a flex with a vertical axis), this aligns the
```

Comment:
I've added some "see also"s to the enum definition, but I'm reluctant to point to [Flex] too strongly.